cypress/integration/e2e/search.spec.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable indent */
+
+/* eslint-disable no-undef */
+describe('Navigate to Gallery', () => {
+  it('Navigate to the Gallery page', () => {
+    cy.visit('http://localhost:9090/rmrk/gallery')
+    cy.wait(2000)
```

Comment:
please don't use wait and instead try to watch for the response of initial graphql call via [cy.intercept](https://docs.cypress.io/api/commands/intercept)